tests/test_cli_compile.py

```diff
@@ -513,40 +512,34 @@ def test_locally_available_editable_package_is_not_archived_in_cache_dir(
 @pytest.mark.parametrize(
     ("line", "dependency"),
     (
-        # zip URL
         # use pip-tools version prior to its use of setuptools_scm,
         # which is incompatible with https: install
-        (
+        pytest.param(
             "https://github.com/jazzband/pip-tools/archive/"
             "7d86c8d3ecd1faa6be11c7ddc6b29a30ffd1dae3.zip",
             "\nclick==",
+            id="Zip URL",
         ),
-        # scm URL
-        (
+        pytest.param(
             "git+git://github.com/jazzband/pip-tools@"
             "7d86c8d3ecd1faa6be11c7ddc6b29a30ffd1dae3",
             "\nclick==",
+            id="VCS URL",
         ),
-        # wheel URL
-        (
+        pytest.param(
             "https://files.pythonhosted.org/packages/06/96/"
             "89872db07ae70770fba97205b0737c17ef013d0d1c790"
             "899c16bb8bac419/pip_tools-3.6.1-py2.py3-none-any.whl",
             "\nclick==",
+            id="Wheel URL",
         ),
-        (
+        pytest.param(
             "pytest-django @ git+git://github.com/pytest-dev/pytest-django"
             "@21492afc88a19d4ca01cd0ac392a5325b14f95c7"
             "#egg=pytest-django",
             "git+git://github.com/pytest-dev/pytest-django"
             "@21492afc88a19d4ca01cd0ac392a5325b14f95c7#egg=pytest-django",
-        ),
-        (
-            "git+git://github.com/open-telemetry/opentelemetry-python.git"
```

Comment:
I removed this test and created a local one with a subdirectory inside `test_data`, here: https://github.com/jazzband/pip-tools/pull/1453/files#diff-99b06d953a9978ee564df2e67547b8a76f3b1ab116ab7c89a9005b9239f2a904R608

I did this to decrease the time it takes to run this test.